[2025-03-24T12:25:04.675+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-24T12:25:04.683+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_pipeline.load_to_hdfs scheduled__2025-03-22T00:00:00+00:00 [queued]>
[2025-03-24T12:25:04.688+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_pipeline.load_to_hdfs scheduled__2025-03-22T00:00:00+00:00 [queued]>
[2025-03-24T12:25:04.688+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 5
[2025-03-24T12:25:04.697+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load_to_hdfs> on 2025-03-22 00:00:00+00:00
[2025-03-24T12:25:04.699+0000] {standard_task_runner.py:72} INFO - Started process 403359 to run task
[2025-03-24T12:25:04.701+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'fraud_detection_pipeline', 'load_to_hdfs', 'scheduled__2025-03-22T00:00:00+00:00', '--job-id', '167', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmpwnllfaah']
[2025-03-24T12:25:04.702+0000] {standard_task_runner.py:105} INFO - Job 167: Subtask load_to_hdfs
[2025-03-24T12:25:04.722+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_pipeline.load_to_hdfs scheduled__2025-03-22T00:00:00+00:00 [running]> on host usmail-ThinkBook-14-G2-ITL
[2025-03-24T12:25:04.756+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ismailsamilacc@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fraud_detection_pipeline' AIRFLOW_CTX_TASK_ID='load_to_hdfs' AIRFLOW_CTX_EXECUTION_DATE='2025-03-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-22T00:00:00+00:00'
[2025-03-24T12:25:04.757+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-24T12:25:06.105+0000] {logging_mixin.py:190} WARNING - creating folder in HDFS
[2025-03-24T12:25:06.105+0000] {dags.py:90} INFO - creating folder in HDFS
[2025-03-24T12:25:06.106+0000] {logging_mixin.py:190} WARNING - Copying data to data into /hadoop_namenode mounted volume
[2025-03-24T12:25:06.106+0000] {dags.py:93} INFO - Copying data to data into /hadoop_namenode mounted volume
[2025-03-24T12:25:07.455+0000] {logging_mixin.py:190} WARNING - Data loaded to HDFS
[2025-03-24T12:25:07.455+0000] {dags.py:96} INFO - Data loaded to HDFS
[2025-03-24T12:25:07.456+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-24T12:25:07.459+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-24T12:25:07.459+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_pipeline, task_id=load_to_hdfs, run_id=scheduled__2025-03-22T00:00:00+00:00, execution_date=20250322T000000, start_date=20250324T122504, end_date=20250324T122507
[2025-03-24T12:25:07.480+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-24T12:25:07.493+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-24T12:25:07.496+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
